src/components/QuizButton.tsx

```diff
@@ -8,10 +9,29 @@ type QuizButtonProps = {
   onAnswerClick: (answer: string) => void;
 };
 
+function shuffleArray(array: string[]) {
+  const shuffledArray = [...array];
+  for (let i = shuffledArray.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [shuffledArray[i], shuffledArray[j]] = [shuffledArray[j], shuffledArray[i]];
+  }
+  return shuffledArray;
+}
```

Comment:
```ts
function shuffleArray<T>(array: T[]): T[] {
    return array
        .map((value) => ({ value, sort: Math.random() }))
        .sort((a, b) => a.sort - b.sort)
        .map(({ value }) => value);
}
//...
shuffleArray<number>([1,2,3,4]);
shuffleArray<string>(['a', 'b', 'c', 'd']);
shuffleArray<{ name: string, age: number }>([
  { name: 'John', age: 15 },
  { name: 'Alice', age: 26 },
  { name: 'Bruce', age: 34 },
]);
```
the declarative implementation is more preferable
and above it is more flexible because of generics